src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -273,10 +256,7 @@ create_reference_table(PG_FUNCTION_ARGS)
 	ObjectAddress tableAddress = { 0 };
 
 	bool viaDeprecatedAPI = false;
-
-	EnsureCoordinator();
-	CheckCitusVersion(ERROR);
-	EnsureTableOwner(relationId);
+	EnsureTableCanBeCreated(relationId);
```

Comment:
very minor: maybe an empty line before the call made to `EnsureTableCanBeCreated`